changelog.md

```diff
@@ -59,11 +60,6 @@ Bug Fixes:
 * Avoid opening `/dev/tty` when `--no-warn` is given.
 * Fixed some typo errors in `README.md`.
 
-Internal:
----------
-
-* Use server info from pymysql connection object.
```

Comment:
This is not internal, it's a user-facing bug.

Reply:
Right. I wrote that line before I knew of the bug.